app/views/banners/_form.html.erb

```diff
@@ -21,9 +21,10 @@
       <%= form.label :name %>
       <%= form.text_field :name, required: true %>
     </div>
-    <div class="form-check checkbox-style mb-20">
-      <%= form.check_box :active, class: 'form-check-input' %>
+    <div data-controller="checkbox reveal" data-action="<%= @has_alternate_active_banner && 'checkbox:toggle->reveal#toggle'%>" data-reveal-hidden-class="d-none" class="form-check checkbox-style mb-20">
+      <%= form.check_box :active, data: { action: "click->checkbox#toggle", checkbox_target: "checkbox" }, class: 'form-check-input' %>
       <%= form.label :active, "Active?", class: 'form-check-label' %>
+      <span data-reveal-target="item" class="text-danger<%= form.object.active && @has_alternate_active_banner ? '' : ' d-none' %>">Warning: This will replace your current active banner</span>
```

Comment:
I don't love a ternary that can return `''` but thought it read a little better than `(!form.object.active || !@has_alternate_active_banner) && ' d-none'`. Any opinion?

Reply:
What do you think about moving it to a helper method? Something like:
```
class="text-danger <%= alternate_banner_class(form.object.active) %>"
```

```
def alternate_banner_class(active)
  unless active && @has_alternate_active_banner
    "d-none"
  end
end
```

Reply:
Bingo! I like that much more. Thanks!